examples/differential_correction_deleonberne.py

```diff
@@ -0,0 +1,224 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jul 30 10:02:48 2019
+
+@author: Wenyang Lyu and Shibabrat Naik
+
+Script to compute unstable periodic orbits at specified energies for the coupled quartic 
+Hamiltonian using differential correction
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.integrate import solve_ivp
+import time
+
+# This needs testing for installation 
+import sys
+import importlib
+sys.path.insert(0, './src/')
+
+import differential_correction
+importlib.reload(differential_correction)
+import differential_correction as diffcorr
+
+import deleonberne_hamiltonian
+importlib.reload(deleonberne_hamiltonian)
+import deleonberne_hamiltonian as deleonberne
+# This needs testing for installation
+
+import matplotlib as mpl
+from matplotlib import cm
+mpl.rcParams['mathtext.fontset'] = 'cm'
+mpl.rcParams['mathtext.rm'] = 'serif'
+
+        
+#%% Setting up parameters and global variables
+    
+save_final_plot = True
+show_final_plot = False
+N = 4          # dimension of phase space
+MASS_A = 8.0 

```

Comment:
Why are these in capital letters?

Reply:
These are constants for a given problem (https://pep8.org/#constants)